builder/builder.go

```diff
@@ -650,20 +651,42 @@ func (b *Builder) buildUpdateWithNewSwupd(timer *stopWatch, mixVersion uint32, m
 	for _, f := range mom.Files {
 		fmt.Printf("- %-20s %d\n", f.Name, f.Version)
 	}
-	if !skipSigning {
+
+	if skipSigning {
+		fmt.Println("SKIPPING manifest signing.")
```

Comment:
Don't need to print this, just remove this if IMHO.

Reply:
Removed this from the commit that added it (and other PR) and from this one.